lib/components/SpecialTextField.dart

```diff
@@ -90,7 +90,7 @@ class _SpecialTextFieldState extends State<SpecialTextField> {
         autofocus: widget.autofocus,
         focusNode: widget.focusNode,
         onChanged: widget.onChanged,
-        enabled: widget.enabled,
+        enabled: widget.enabled ?? false, // TODO: maybe there's a better way to do this?  widget.enabled is nullable, should it be?
```

Comment:
probably not - the only reason i can think of for it being nullable is if you wanted enabled to be the default state when not specified (but this code defaults to false when unspecified)